satpy/readers/seviri_l1b_hrit.py

```diff
@@ -725,8 +739,20 @@ def _update_attrs(self, res, info):
         res.attrs['standard_name'] = info['standard_name']
         res.attrs['platform_name'] = self.platform_name
         res.attrs['sensor'] = 'seviri'
-        res.attrs['nominal_start_time'] = self.nominal_start_time
-        res.attrs['nominal_end_time'] = self.nominal_end_time
+        res.attrs['nominal_start_time'] = """
+        Deprecation warning: nominal_start_time should be accessed via the time_parameters attrs
+        nominal_start_time is also available directly via start_time
+        """
+        res.attrs['nominal_end_time'] = """
+        deprecation warning: nominal_end_time should be accessed via the time_parameters attrs
+        nominal_end_time is also available directly via end_time
+        """
+        res.attrs['time_parameters'] = {
+            'nominal_start_time': self.nominal_start_time,
+            'nominal_end_time': self.nominal_end_time,
+            'observation_start_time': self.observation_start_time,
+            'observation_end_time': self.observation_end_time,
+            }
```

Comment:
This is not really a deprecation warning, but it might actually break users' applications.

I'm not sure how/if we added deprecation warnings for attributes in the past. Alternatively we could keep the attribute as is and just add a deprecation note to the documentation.

Reply:
This these attribute  only exist for the seviri_hrit reader(and not the native nor nc ones), I will revert the change on the nominal start/end time attributes to the former interface and add a deprecation warning in the doc string of this reader only. 
Let me know if you think it should be done some other way?

Reply:
see 
https://github.com/pytroll/satpy/pull/2420/commits/46eb8f0f2ac8e0f5d62563775da1b2f118d6fb9e
and 
https://github.com/pytroll/satpy/pull/2420/commits/d77d2f2ac49cd01f2df2247b1afcaf1bc6b9a691